app/controllers/questions_controller.rb

```diff
@@ -15,40 +15,57 @@ def index
     render json: questions
   end
 
+  def user_questions
+    questions = @current_user.questions
+    if questions.empty?
+      render json: [], status: :ok
+    else
+      render json: questions, status: :ok
+    end
+  end
```

Comment:
If there are no questions created by the user then @current_user.questions will return an empty array. So, You can just return the response you get from the @current_user.questions. 

At line number 20, You're checking if the questions.empty? which means, "Is the question array empty?". If yes, return an empy array otherwise return questions. 

But you can simply return the questions from there.

Reply:
hhmm right